src/storage/mod.rs

```diff
@@ -324,12 +324,12 @@ mod test {
                 "text/rust"
             );
 
-            let file = backend.get("rustdoc/Cargo.toml").unwrap();
+            let file = backend.get("rustdoc/Cargo.toml", std::usize::MAX).unwrap();
             assert_eq!(file.content, b"data");
             assert_eq!(file.mime, "text/toml");
             assert_eq!(file.path, "rustdoc/Cargo.toml");
 
-            let file = backend.get("rustdoc/src/main.rs").unwrap();
+            let file = backend.get("rustdoc/src/main.rs", std::usize::MAX).unwrap();
```

Comment:
I think these tests could be improved by setting the limits to under, over and exactly at the length of the content we give, that way we can ensure that the buffering works with actual files as well

Reply:
Hmm, these tests are not testing the size limits, they're just the older tests I had to change due to the new argument.